tests/xfstests/run.pm

```diff
@@ -315,6 +209,27 @@ END_CMD
     }
 }
 
+sub reload_loop_device {
+    assert_script_run("losetup -fP $INST_DIR/test_dev");
+    my $scratch_amount = script_output("ls $INST_DIR/scratch_dev* | wc -l");
+    my $scratch_num    = 1;
+    while ($scratch_amount >= $scratch_num) {
+        assert_script_run("losetup -fP $INST_DIR/scratch_dev$scratch_num", 300);
+        $scratch_num += 1;
+    }
+    script_run('losetup -a');
+    format_partition("$INST_DIR/test_dev", $FSTYPE);
+}
+
+# Umount TEST_DEV and SCRATCH_DEV
+sub umount_xfstests_dev {
+    script_run('umount ' . get_var('XFSTESTS_TEST_DEV') . ' &> /dev/null')    if get_var('XFSTESTS_TEST_DEV');
+    script_run('umount ' . get_var('XFSTESTS_SCRATCH_DEV') . ' &> /dev/null') if get_var('XFSTESTS_SCRATCH_DEV');
+    if (get_var('XFSTESTS_SCRATCH_DEV_POOL')) {
+        script_run("umount $_ &> /dev/null") foreach (split ' ', get_var('XFSTESTS_SCRATCH_DEV_POOL'));
```

Comment:
something like this shouldn't fail. How about `assert_script_run` instead? Same question applies to other calls to `script_run` as well

Reply:
Yes, but I don't want add any additional changes, everything is added/changed to use serial console.

Reply:
Sorry, I don't understand. If you say you just move code then of course it is ok to keep that. I am just saying that by default `assert_script_run` should be used unless the exit code of `script_run` is explicitly handled.

Reply:
Yes `assert_script_run` should be used, I just don't want add unnecessary changes.